### ISSUE TITLE
Update AntiXSS library

### ISSUE THREAD

**Issue by tsmgeek** [salesagility/SuiteCRM#9240]
https://raw.githubusercontent.com/voku/anti-xss/

Can we update AntiXSS library to latest as its doing some odd things such as stripping out `data` uri elements from email messages 
The updated version does not apply a general regex.
We do not need AntiXSS to do this if we are using HTMLPurifier.

**Comment by Dillon-Brown** [comment]
Agreed, we should update this to latest every release. As for HTMLPurifier.... Agreed it's redundant to have both but at the moment we do XSS cleaning on every request with a regex + AntiXSS -> clean again with HTMLPurifier depending on the field/module -> clean again with custom logic depending on the class/method.

It's pretty ridiculous but not exactly easy to solve. Ideally I'd want to remove the AntiXSS method and only call HTMLPurifier when actually required but that would require tracking down every possible XSS entrypoint or we could be introducing a ton of security issues. Thankfully this will be handled much much better in Suite-8.

**Comment by tsmgeek** [comment]
I understand, at least if we keep with the latest versions within the 7.x series then we will not have odd side cases that took a while for me to track down.
